src/test/java/org/openrewrite/java/testing/jmockit/JMockitMockUpToMockitoTest.java

```diff
@@ -0,0 +1,369 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.jmockit;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.java.testing.jmockit.JMockitTestUtils.setParserSettings;
+
+class JMockitMockUpToMockitoTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        setParserSettings(spec, "jmockit-1.22", "junit-4.13.2");
+    }
+
+    @DocumentExample
+    @Test
+    void mockUpStaticMethodTest() {
+        //language=java
+        rewriteRun(
+          java(
+                """
+            import mockit.Mock;
+            import mockit.MockUp;
+            import static org.junit.Assert.assertEquals;
+            
+            class MockUpTest {
+                void test() {
+                    new MockUp<MyClazz>() {
+            
+                        @Mock
+                        public int staticMethod() {
+                            return 1024;
+                        }
+            
+                        @Mock
+                        public int staticMethod(int v) {
+                            return 128;
+                        }
+                    };
+                    assertEquals(1024, MyClazz.staticMethod());
+                    assertEquals(128, MyClazz.staticMethod(0));
+                }
+            
+                public static class MyClazz {
+                    public static int staticMethod() {
+                        return 0;
+                    }
+            
+                    public static int staticMethod(int v) {
+                        return 1;
+                    }
+                }
+            }
+            """, """
+            import org.mockito.MockedStatic;
+            
+            import static org.junit.Assert.assertEquals;
+            import static org.mockito.Mockito.*;
+            
+            class MockUpTest {
+                void test() {
+                    MockedStatic<MockUpTest.MyClazz> mockStaticMockUpTest_MyClazz = mockStatic(MockUpTest.MyClazz.class);
```

Comment:
Put this in a try with resources so that if anything generates and exception, the close will definitely be called, otherwise can cause test failures

Reply:
Using try with resources may have a big effect the structure by enclosing codes inside. Don't feel like a good solution.